core/amber/src/main/scala/edu/uci/ics/texera/workflow/WorkflowCompiler.scala

```diff
@@ -222,13 +214,11 @@ class WorkflowCompiler(
             } else {
               None
             }
-          sink.setStorage(
-            storage.create(
-              s"${o.getContext.executionId}_",
-              storageKey,
-              storageType,
-              sinkStorageSchema
-            )
+          storage.create(
+            s"${o.getContext.executionId}_",
```

Comment:
Can we have a util function to generate the storageKey? Otherwise each creator/reader of the storage needs to build the key by themselves. `ResultStorage` might be a good place to put this function.

Reply:
currently, the storage key is just the upstream operator id. I will open another PR to add the port id to the storage key.